src/test/java/hw3/pages/components/DifferentElementsInputFieldsComponent.java

```diff
@@ -0,0 +1,41 @@
+package hw3.pages.components;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class DifferentElementsInputFieldsComponent extends AbstractComponent {
+    @FindBy(xpath = "//label[normalize-space()='Water']")
+    WebElement water;
+
+    @FindBy(xpath = "//label[normalize-space()='Wind']")
+    WebElement wind;
+
+    @FindBy(xpath = "//label[normalize-space()='Selen']")
+    WebElement selen;
+
+    @FindBy(css = ".colors")
+    WebElement colors;
+
+    @FindBy(xpath = "//*[text()='Yellow']")
+    WebElement yellow;
```

Comment:
I suppose that you do not need this two locator. Replace them to the one to use special element for the dropdown in Selenium